### DISCUSSION TITLE
Hotkeys to increase and decrease Volume and to mute

### DISCUSSION THREAD

**Discussion by Apollosport** [Belphemur/SoundSwitch#1063] [FAQ]
Hey there,
I would love to see a simple functionality to change the output levels of the audio device with hotkeys. An additional hotkey to mute would be great.

Regards

**Comment by asamoylenko8000** [comment]
I need also this feature. It would be really nice.

**Comment by Smashthedude** [comment]
Hotkeys for volume up/down would be awesome. There's an app for that but it keeps crashing and it would be nice to have such functionality into SoundSwitch. For example, in the other software i was using right click + Scroll up for volume up and rclick+scrldown for volume down. it's a really handy feature, i bet many will use it. Cheers.
